### PR TITLE
Improve styling of buttons

### PR DESCRIPTION
**Pull request by falbru**
**before**
image

**after**
image

Just a proof of concept. The colors can be changed later as we add more games and songs.

### CODE REVIEW

File: src/components/ui/nav-button.tsx [hunk 1]
```diff
@@ -5,16 +5,28 @@ import Link from "next/link";
 interface NavButtonProps {
   label: string;
   href: string;
+  color?: string;
```
Comment (Bestem0r) [inline src/components/ui/nav-button.tsx:8]:
```suggestion
  color?: "blue" | "red" | "green" | "fuchsia" | "teal" | "orange";
```

File: src/components/ui/button.tsx [hunk 2]
```diff
@@ -40,11 +40,14 @@ export interface ButtonProps
...
 const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
-  ({ className, variant, size, asChild = false, ...props }, ref) => {
+  ({ color, className, variant, size, asChild = false, ...props }, ref) => {
     const Comp = asChild ? Slot : "button";
     return (
       <Comp
-        className={cn(buttonVariants({ variant, size, className }))}
+        className={cn(
+          buttonVariants({ variant, size, className }),
+          color && `${color} hover:${color}/90`,
```
Comment (Bestem0r) [inline src/components/ui/button.tsx:49]: Is this used?

File: src/components/ui/nav-button.tsx [hunk 3]
```diff
@@ -5,16 +5,28 @@ import Link from "next/link";
...
+    orange: "bg-orange-500 hover:bg-orange-500/90",
+  };
+
   return (
     <Link href={href} passHref>
-      <Button className="w-full group" size="lg">
-        {icon && icon}
-        {label}
+      <Button className={`w-full group ${colorVariants[color!]}`} size="lg">
+        <div className="flex gap-1 items-center">
```
Comment (Bestem0r) [inline src/components/ui/nav-button.tsx:26]:
```suggestion
        <div className="flex gap-4 items-center">
```

**Review by Bestem0r — APPROVED** [review]
Great! The icons and the colors look real good. While we're on it, could we perhaps also make the buttons bigger? I suggest `lg: "h-14 rounded-md px-6",`. What do you think?

File: src/components/ui/button.tsx (hunk 2 above)
Comment (falbru) [reply src/components/ui/button.tsx:49]: Nope haha. Good catch!

**Review by Bestem0r — APPROVED** [review]
Looks good!

**Pull request merged**